core/storage/user/gae_models.py

```diff
@@ -683,23 +681,22 @@ def get(cls, model_id):
             model_id: str. The model id of the user and the skill.
 
         Returns:
-            list. All skill mastery models where user_id belongs to the
-                entered user are returned as a list where each element
-                is one skill mastery model.
+            dict. Skill mastery model with id equal to model_id.
```

Comment:
It's not a dict. Please double-check the type.

Do you even need this method, btw? I think it's already defined in the base class (see base_models.py), and somewhat more robustly.

Reply:
(Also, if you do decide to drop this method, it's still a good idea to keep the tests in place, I think.)

Reply:
Removed the method and use that defined in BaseClass

Reply:
Same thing as this [comment](https://github.com/oppia/oppia/pull/4997#discussion_r194220807).
I think it's safe to keep considering any false request